### PR TITLE
Add configuration properties

### PR DESCRIPTION
**Pull request by onecreative**

### CODE REVIEW

File: marketplace_builder/views/pages/reference/form-configurations/form-configurations.liquid [hunk 1]
```diff
@@ -9,6 +9,26 @@ searchable: true
...
+| `callback_actions`       |             |         |         | No       |
+| `configuration`          |             |         |         | No       |
+| `default_payload`        |             |         |         | No       |
+| `email_notifications`    |             |         |         | No       |
+| `flash_alert`            |             |         |         | No       |
+| `flash_notice`           |             |         |         | No       |
+| `redirect_to`            |             |         |         | No       |
+| `resource_owner`         |             |         | `self`  | No       |
+| `sms_notifications`      |             |         |         | No       |
+| `spam_protection`        |             |         |         | No       |
```
Comment (pavelloz) [inline marketplace_builder/views/pages/reference/form-configurations/form-configurations.liquid:30]: @Slashek spam_protection isnt supported anymore, right?

**Comment by pavelloz** [comment]
@onecreative duplicate of #172 ?

File: marketplace_builder/views/pages/reference/form-configurations/form-configurations.liquid (hunk 1 above)
Comment (Slashek) [reply marketplace_builder/views/pages/reference/form-configurations/form-configurations.liquid:30]: it is - recaptcha :)

**Comment by onecreative** [comment]
@pavelloz Yes, sorry about that. Definitely use 172, as it has more info on it.

**Pull request closed without merging**